app/views/info/_vendor_terms_text.html.erb

```diff
@@ -112,7 +119,7 @@
 <h3>
 <a name="vendor-registration" class="anchor" href="#vendor-registration"><span class="mini-icon mini-icon-link"></span></a>Vendor Registration</h3>
 
-<p>The Bike Index Vendor Service is only made available under this Agreement to persons in the United States that operate a business selling goods or services, or operate cycling related organizations. To use Bike Index to register bikes, you will first have to register for a Vendor Account. When you register for a Bike Index Vendor Account, we will collect basic information including your name, company name, location, email address, and phone number. If you have not already done so, you will also be required to provide an email address and password for your Bike Index account.</p>
+<p>The Bike Index Vendor Service is only made available under this Agreement to persons in the United States that operate a business selling goods or services, or operate cycling related organizations. To use Bike Index to register bikes, you will first have to register for a "Vendor Account." When you register for a Bike Index Vendor Account, we will collect basic information including your name, company name, location, email address, and phone number. If you have not already done so, you will also be required to provide an email address and password for your Bike Index account.</p>
```

Comment:
> only made available under this Agreement to persons in the United States

Is this boilerplate language or some sort of legal thing? Cycling is pretty big in other countries outside the US.